### ISSUE TITLE
Isort Compatiblity Black [Pre-Commit]

### ISSUE THREAD

**Issue by AxanIqbal** [cookiecutter/cookiecutter-django#4170]
## Description

I think we should have set the profile mode of isort to Black: [Ref](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html)

## Rationale

I had a lot of problem with github actions and git hook every time i have to check details. Fixed it with that config hope that help for others

**Comment by BenAkko** [comment]
How did you integrate this into your project, if you don't mind me asking?

**Comment by AxanIqbal** [comment]
> How did you integrate this into your project, if you don't mind me asking?

you just have to put `profile = black` in last line of `[isort]` in setup.cfg

```cfg
[isort]
line_length = 88
known_first_party = muslim_love,config
multi_line_output = 3
default_section = THIRDPARTY
skip = venv/
skip_glob = **/migrations/*.py
include_trailing_comma = true
force_grid_wrap = 0
use_parentheses = true
profile = black
```

**Comment by BenAkko** [comment]
Thanks for your reply, much appreciated.

**Comment by priyank-panchal** [comment]
Hi, I am just starting to contribute to open source, and I believe this issue would be a good fit for me. Can I take it up?

**Comment by foarsitter** [comment]
@priyank-panchal asking permission to fix an issue isn't necessary. Looking forward to your pull request.

**Comment by browniebroke** [comment]
Actually, that should be fixed since #4276
